### ISSUE TITLE
Missing cancel button on show/hide options modal

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#4921]
For accessibility and consistency all modals should have a cancel button. It looks like this one was missed.

<img width="571" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1ZTQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--678f87d55b0a58f20ce2310e40ac31d6aaad75a5/image.png" alt="image.png" />
